lib/MusicBrainz/Server/Entity/AliasType.tt

```diff
@@ -0,0 +1,31 @@
+[%# It's correct to make changes to this file. The perl comment below is for
+    the benefit of people looking at the auto-generated files. To regenerate
+    them from this template, run ./script/generate_code.pl. -%]
+# Automatically generated, do not edit.
+package MusicBrainz::Server::Entity::[% model %];
+
+use Moose;
+
+extends 'MusicBrainz::Server::Entity::AliasType';
+
+with 'MusicBrainz::Server::Entity::Role::OptionsTree' => {
+    type => '[% model %]',
+};
+
+sub entity_type { '[% alias_type %]' }
+
+__PACKAGE__->meta->make_immutable;
+
+no Moose;
+
+1;
+
+=head1 COPYRIGHT AND LICENSE
+
+Copyright (C) 2012 MetaBrainz Foundation
```

Comment:
2023?

Reply:
The base code was pretty much all copied from lib/MusicBrainz/Server/Entity/AliasType.pm which has the same copyright, but I suppose we could update it for the template changes.